kolibri/core/assets/src/views/k-checkbox/index.vue

```diff
@@ -137,6 +137,10 @@
           this.$emit('change', this.isCurrentlyChecked, event);
         }
       },
+      markInactive() {
+        this.isActive = false;
+        this.$emit('blur');
```

Comment:
Add event description.

Reply:
:pray: :) (I need to sound nicer.)

Reply:
:smirk: thanks lol

What're these for? Do they get included in documentation?

Reply:
For the style guide documentation.